src/XrdClLazyDownload.cc

```diff
@@ -107,7 +107,16 @@ LDFile::Close(XrdCl::ResponseHandler *handler,
     AtomicBeg(m_mutex);
     AtomicCAS(m_is_open, true, false);
     AtomicEnd(m_mutex);
-    return m_fh.Close(handler, timeout);
+    if (m_fh.IsOpen()) {
```

Comment:
I don't think this is thread-safe.  Shouldn't we be doing something based on m_is_open?

I'm worried about giving the correct error code in case the user does close() twice.